RocketChat/Rocket.Chat
# fix: Prevent refetch queries on windows focus

## Proposed changes (including videos or screenshots)



Some request was refetching multiple times causing http errors and loading several times without reason

Added as a default behavior of `QueryClient()` from `@tanstack/react-query` the option to NOT refetch on windows Focus to improve performance and avoid some bugs.

There were some files that implemented this option in some `useQuery` implementation manually, since it's a default behavior now I removed this option from several queries.




## Issue(s)


## Steps to test or reproduce

1. Go to RC client
2. in the main screen open the Network tab
3. Click on the window and outside the window (e.g: Browser Bookmark bar or tab bar) 
4. Check some queries being refetched over and over again  

## Further comments


[WM-144](https://rocketchat.atlassian.net/browse/WM-144)


[WM-144]: https://rocketchat.atlassian.net/browse/WM-144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
